generative_ai/embeddings/document_retrieval_example.py

```diff
@@ -13,23 +13,25 @@
 # limitations under the License.
 
 # [START generativeaionvertexai_embedding]
-from typing import List, Optional
+from __future__ import annotations
 
 from vertexai.language_models import TextEmbeddingInput, TextEmbeddingModel
 
 
 def embed_text(
-    texts: list = None,
-    task: str = "RETRIEVAL_DOCUMENT",
-    dimensionality: Optional[int] = 256,
-) -> List[List[float]]:
+    texts: list[str] | None = None,
```

Comment:
@Valeriy-Burlaka - Can we improve this function to have no arguments?

If need, you can define these variables inside the function itself. More details @ go/devrel-hidden-variables